LizarroAnghela/Collections/src/Circles.java

```diff
@@ -0,0 +1,32 @@
+public class Circles {
```

Comment:
Classes represent a singular object abstraction of the real life.
That's why class names have singular names.
Change Circles to Circle please.